django/template/base.py

```diff
@@ -278,19 +284,18 @@ def create_token(self, token_string, in_tag):
                 self.verbatim = False
         if in_tag and not self.verbatim:
             if token_string.startswith(VARIABLE_TAG_START):
-                token = Token(TOKEN_VAR, token_string[2:-2].strip())
+                token = Token(TOKEN_VAR, token_string[2:-2].strip(), position, self.lineno)
```

Comment:
Is there a good reason to pass `position` explicitly in argument and `lineno` implicitly through the lexer's state? That looks odd. Can you pass `lineno` explicitly as well?